lib/src/components/NxThreatCounter/__tests__/NxThreatCounter.test.tsx

```diff
@@ -13,41 +13,149 @@ describe('NxThreatCounter', function() {
         criticalCount: 66,
```

Comment:
Now that these are all optional, I'd say the `minimalProps` should be an empty object.  This would simplify many of the tests below.

Reply:
> Now that these are all optional, I'd say the `minimalProps` should be an empty object. This would simplify many of the tests below.

Done in https://github.com/sonatype/sonatype-react-shared-components/pull/418/commits/8cce8f0e7e32b56afa23137b9a7b1a8cb948a2bd